src/backfill/prioritizer.ts

```diff
@@ -46,10 +47,18 @@ export class Prioritizer implements StepPrioritizer<JobId, JobState> {
 		const repo = jobState.repository;
 
 		if (Prioritizer.hasWaitingPullrequests(jobState.repository)) {
-			if (!repo.lastPullCursor) {
-				repo.lastPullCursor = 0;
+			if (!repo.pullCursor) {
+				repo.pullStatus = "failed";
+			} else {
+				const cursorParts = repo.pullCursor.match(this.cursorRegex);
+				if (!cursorParts || cursorParts.length != 3) {
+					repo.pullStatus = "failed";
+				} else {
+					const cursorId = cursorParts[1];
+					const index = cursorParts[2];
+					repo.pullCursor = `${cursorId} ${+index + this.pullrequestSkipCount}`;
+				}
```

Comment:
I agree, it's taken from below and is a horrible standard, but as far as I can tell, this file isn't being used but I had to change it because the old pullCursor was for numbers and not strings.

I think the idea is to remove this file eventually or just fix our  backfilling mechanism